spec.bs

```diff
@@ -1449,19 +1449,14 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo| and [=
   1. Set |config| to |leadingBidInfo|'s [=leading bid info/auction config=].
   1. Let |sellerReportingScript| be the result of [=fetching script=] with |config|'s
     [=auction config/decision logic url=].
-  1. Let |reportResultOutputJS| be the result of [=evaluating a reporting script=] with
-     |sellerReportingScript|, "`reportResult`", and « |config|, |browserSignals|».
+  1. Let « |sellerSignals|, |reportUrl|, |reportingBeaconMap| » be the result of

```

Comment:
Do we have to do anything special here to handle the case where these tree list items are null?

Reply:
sellerSignals will not be null. Our implementation has it as null, and change it to string "null" when passing to `reportWin()`, and has a [comment](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:content/browser/interest_group/interest_group_auction_reporter.cc;drc=3e67c0491457f1a05fb7816346fc6d9676bb7a16;l=514) about making it non-optional, so I just updated here to let `reportResult()` return "null" instead of null.

The other two are used for fenced frame reporting that Garrett is working on. So that'll be handled there.

Reply:
OK thanks